tests/Concerns/WithCountTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Maatwebsite\Excel\Tests\Concerns;
+
+use Maatwebsite\Excel\Tests\Data\Stubs\AfterQueueExportJob;
+use Maatwebsite\Excel\Tests\Data\Stubs\Database\Group;
+use Maatwebsite\Excel\Tests\Data\Stubs\Database\User;
+use Maatwebsite\Excel\Tests\Data\Stubs\FromQueryWithCount;
+use Maatwebsite\Excel\Tests\TestCase;
+
+class WithCountTest extends TestCase
+{
+    /**
+     * Setup the test environment.
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->loadLaravelMigrations(['--database' => 'testing']);
+        $this->loadMigrationsFrom(dirname(__DIR__) . '/Data/Stubs/Database/Migrations');
```

Comment:
This second one is not necessary I think. 
See example of working database test: https://github.com/Maatwebsite/Laravel-Excel/blob/3.0/tests/QueuedQueryExportTest.php

Reply:
This particular test case, needs:

- the users table (created by the call to `loadLaravelMigrations`) and
- two more tables (groups and a group_user pivot table) created by the custom migrations loaded by the call to `loadMigrationsFrom`.

Reply:
Hm, can you have a look at why the test is failing on Travis then?

Reply:
The new test is no longer failing, but I do see some failures in some unrelated test. Tests that check the Content-Disposition header. File names are not enclosed in quotes in 5.6 but they are in 5.5.

Do we need a new PR to tackle this, though?